scripts/roomba.py

```diff
@@ -106,6 +106,10 @@ def __init__(self, namespace):
                                           Twist,
                                           queue_size=10)
 
+        self._state_publisher = rospy.Publisher('{}state'.format(namespace),
```

Comment:
This topic needs a better name, like not_moving or something

Reply:
`turning` okay?

Reply:
changed to turning